.github/workflows/delete-closed-pr-docs.yaml

```diff
@@ -6,7 +6,15 @@ on:
   workflow_dispatch:
 
 jobs:
+  prevent-running-on-forks:
+    name: Don't run scheduled jobs on forks
+    runs-on: ubuntu-latest
+    if: ${{ (github.event_name != 'schedule') || (github.event_name == 'schedule' && github.repository == 'autowarefoundation/autoware.universe') }}
+    steps:
+      - run: "true"
+
   delete-closed-pr-docs:
+    needs: prevent-running-on-forks
```

Comment:
Seeing this error, I feel we should fix the workflow-side.
https://github.com/kenji-miyake/autoware.universe/runs/5189322226?check_suite_focus=true

Reply:
https://github.com/autowarefoundation/autoware-github-actions/issues/79

Reply:
Resolved: https://github.com/autowarefoundation/autoware-github-actions/pull/112

Reply:
I believe you can revert the changes to this file.